src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Apprentice/DetailsViewModelMapper.cs

```diff
@@ -22,8 +22,26 @@ public DetailsViewModelMapper(ICommitmentsApiClient commitmentApiClient, IEncodi
         public async Task<DetailsViewModel> Map(DetailsRequest source)
         {
             var detailsResponseTask = _commitmentApiClient.GetApprenticeship(source.ApprenticeshipId);
-            var priceEpisodes = await _commitmentApiClient.GetPriceEpisodes(source.ApprenticeshipId);
+            var priceEpisodesTask = _commitmentApiClient.GetPriceEpisodes(source.ApprenticeshipId);
+            var pendingUpdatesTask =  _commitmentApiClient.GetApprenticeshipUpdates(new CommitmentsV2.Api.Types.Requests.GetApprenticeshipUpdateRequest
+            {
+                ApprenticeshipId = source.ApprenticeshipId,
+                Status = ApprenticeshipUpdateStatus.Pending
+            });
+
+            await Task.WhenAll(detailsResponseTask, priceEpisodesTask, pendingUpdatesTask);
+
             var detailsResponse = await detailsResponseTask;
+            var priceEpisodes = await priceEpisodesTask;
+            var pendingUpdates = await pendingUpdatesTask;   
+
+            var pendingUpdatesOnApprentice = pendingUpdates.ApprenticeshipUpdates.Count > 0;
```

Comment:
We should change this line to:

```
            var pendingUpdatesOnApprentice =
                pendingUpdates.ApprenticeshipUpdates.Any(x => x.OriginatingParty == Party.Provider);
```

This will filter out any Employer-originated updates, which are covered by another upcoming story. We could rename the variable too.